public/docs/getting_started/client.md

```diff
@@ -4,11 +4,24 @@ There are many Ethereum clients to choose from. We recommend different clients d
 
 ## When developing
 
-### Truffle Develop
+### Ganache
+
+We recommend [Ganache](/ganache), a one-click GUI blockchain you can use for development. It is designed to be interoperable with Truffle. It processes transactions instantly instead of waiting for the default block time -- so you can test that your code works quickly -- and it tells you immediately when your smart contracts run into errors. It also makes a great client for automated testing, and Truffle knows how to use its special features to speed up test runtime by almost 90% compared to other clients.
+
+Ganache, when launched runs on `http://localhost:7545`. It will display the first 10 accounts and the mnemonic used to create those accounts. ([Read more about account mnemonics](https://github.com/bitcoin/bips/blob/master/bip-0039.mediawiki).)
 
-We recommend using Truffle Develop, a development blockchain built directly into Truffle. It processes transactions instantly instead of waiting for the default block time -- so you can test that your code works quickly -- and it tells you immediately when your smart contracts run into errors. It also makes a great client for automated testing, and Truffle knows how to use its special features to speed up test runtime by almost 90% compared to other clients.
+By default, Ganache will use the following mnemonic:
+
+```
+candy maple cake sugar pudding cream honey rich smooth crumble sweet treat
+```
+
+This mnemonic can be changed to be randomly generated, or you can input your own.
+
+
+### Truffle Develop
 
-Because Truffle Develop is built into Truffle, no external installation is required. To run Truffle Develop, type the following into a terminal:
+We also recommend using Truffle Develop, a development blockchain built directly into Truffle. It has many of the same features as Ganache, but in addition is built into Truffle, with no external installation required. To run Truffle Develop, type the following into a terminal:
```

Comment:
You say "built into truffle" / "built directly into Truffle" twice. We should change this. We also shouldn't compare Truffle Develop to Ganache, in that Truffle Develop as no UI, and if we say it's "like Ganache" they may be expecting something entirely different. 

I'd sell Truffle Develop like this: 

> We also recommend using Truffle Develop, a development blockchain built directly into Truffle. Truffle Develop helps you set up an integrated blockchain environment with a single command, no installation required. Run Truffle Develop by typing the following into a terminal: